publication/src/main/scala/hmda/publication/reports/disclosure/DisclosureReports.scala

```diff
@@ -33,6 +33,8 @@ class DisclosureReports(val sys: ActorSystem, val mat: ActorMaterializer) extend
     d51F.map { d51 =>
       println(d51.toJson.prettyPrint)
     }
+
+    //val d53F = D53.generate(larSource, fipsCode, respId, institutionNameF)
```

Comment:
I know we're not actually generating these reports yet, but just out of curiosity, why is this commented out?

Reply:
tl;dr It seemed pointless to include it. ¯\\\_(ツ)_/¯

I don't want to add any more `println`s to the project, but if I uncomment that line as-is, it would change the return type of the `generateReports` method... and it seemed pointless to wrestle with that since this code isn't hooked up anyway.